### PR TITLE
Feature1 Added

### PR DESCRIPTION
**Pull request by rajeshrajkr4**
I have created a new feature named feature1

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is File 1 from feature1 branch
```
Comment (rahul37865) [inline file1.txt:1]: It will be good if you use lorem text

File: file2.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <h1>This is file 2 from feature1 branch</h1>
```
Comment (rahul37865) [inline file2.html:12]: also include at least one h2 below h1

File: file2.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (rahul37865) [inline file2.html:8]: Change Title

**Review by rahul37865 — CHANGES_REQUESTED** [review]

File: file1.txt (hunk 1 above)
Comment (rajeshrajkr4) [reply file1.txt]: yes i added 2 lrem lines

File: file2.html (hunk 3 above)
Comment (rajeshrajkr4) [reply file2.html]: title changed

**Review by rahul37865 — APPROVED** [review]

**Pull request merged**